### PR TITLE
RDK-38254 : Added Telemetry.uploadReport call.

### PR DESCRIPTION
**Pull request by sborushevsky**

### CODE REVIEW

File: Telemetry/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.0] - 2022-10-21
+### Added
+- Added uploadLogs call
```
Comment (anand-ky) [inline Telemetry/CHANGELOG.md:21]: Pls add description for the API. We could include the summary from Telemetry.json.

**Comment by anand-ky** [comment]
@sborushevsky - Pls also add unit tests for the new API/Event.

File: Telemetry/CMakeLists.txt [hunk 2]
```diff
@@ -18,7 +18,6 @@
 set(PLUGIN_NAME Telemetry)
 set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
 
-set(PLUGIN_TELEMETRY_STARTUPORDER "" CACHE STRING "To configure startup order of Telemetry plugin")
```
Comment (npoltorapavlo) [inline Telemetry/CMakeLists.txt:21]: why is this removed

it's used in config

**Review by npoltorapavlo — CHANGES_REQUESTED** [review]

File: Telemetry/Telemetry.cpp [hunk 3]
```diff
@@ -216,6 +236,45 @@ namespace WPEFramework
...
+                LOGERR("%s", str.str().c_str());
+                response["message"] = str.str();
+                returnResponse(false);
+            }
+#else
+            LOGERR("No RBus support");
+            response["message"] = "No RBus support";
+            returnResponse(false);
+#endif
+            returnResponse(true);
```
Comment (npoltorapavlo) [inline Telemetry/Telemetry.cpp:275]: new apis should follow JSON RPC spec https://www.jsonrpc.org/specification#response_object

fyi in case of error no response object is sent

please return error code and document it in the schema

**Review by npoltorapavlo — CHANGES_REQUESTED** [review]

File: Telemetry/Telemetry.json [hunk 4]
```diff
@@ -58,7 +58,13 @@
...
             },
             "result": {
-                "$ref": "#/common/result"        
+                "$ref": "#/common/result"
+            }
+        },
+        "uploadReport": {
+            "summary": "Triggers processing and uploading of telemetry report for legacy Xconf based configuration",
+            "result": {
+                "$ref": "#/common/result"
```
Comment (npoltorapavlo) [inline Telemetry/Telemetry.json:67]: please do not use XRE Service manager object "#/common/result"

in Thunder there are error codes

e.g.

https://github.com/rdkcentral/rdkservices/blob/609ca41faf9b872b4deddf7bec23c8fde8aec4b4/AVInput/AVInput.json#L171

**Review by npoltorapavlo — CHANGES_REQUESTED** [review]

**Review by anand-ky — APPROVED** [review]

**Review by anand-ky — APPROVED** [review]
